mcp-server.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+# Get the directory where this script is located
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+# Run the MCP server from the correct directory
+cd "$SCRIPT_DIR" || exit 1
+exec npx ts-node --transpileOnly src/main.ts "$@"
```

Comment:
Can `mcp-server.sh` and the command be consolidated into `scripts/mcp-server.js` or something similar?